### ISSUE TITLE
[Wav2Vec2.0] Seeking advice on Pre-training

### ISSUE THREAD

**Issue by LEECHOONGHO** [pytorch/fairseq#3842]
![image](https://user-images.githubusercontent.com/44384060/132075364-2b74cac1-0e15-41a2-8bd5-870b3111debd.png)
I am pre-training Wav2Vec2.0, but the loss is not significantly reduced as shown in the graph above and the training is seems to be stucked at contrastive loss 2.4 and diversity loss 0.71 in epoch 7

Is it Normal or should I stop this and fix?

I'm using RTX 3090
batch size 64
Learning Rate : start from 1e-4 to 1e-7 decreasing linearly after first epoch
![image](https://user-images.githubusercontent.com/44384060/132075804-bdd6765d-ff18-47c2-91c6-267b5c3436fb.png)
Data : Noisy 1~6sec utterences like above
temperature : 2 to temperature = max(temperature * 0.999995, 0.5) for every steps

num_codevectors_per_group= 320,
num_codevector_groups = 2,
diversity_loss_weight = 0.1,
masking_rate = 0.1,
distractors_rate = 0.8

Any help Would be much appreciated.

**Comment by RuABraun** [comment]
It's normal. On a single GPU training will take months.

**Comment by LEECHOONGHO** [comment]
Thank you for your advice! I hope using smaller conformer model, and dataset could reduce time.
